models/tridimensional/docking_validation/config.py

```diff
@@ -0,0 +1,89 @@
+import os
+
+
+DIR_BASE = "~/working/"  # this should be changed on windows systems
+
+# ====================================================================== #
+# Directory layout                                                       #
+# ====================================================================== #
+#      /--- PDB ---/   pdb files (protein-dna structure) go here         #
+# BASE /- RESULTS -/   results and simulation scores go here             #
+#      /- SCRIPTS -/   all scripts go here (should be flat)              #
+# ====================================================================== #
+
+DIR_PDB = os.path.join(DIR_BASE, "pdb")
+DIR_RESULTS = os.path.join(DIR_BASE, "results")
+DIR_SCRIPTS = os.path.join(DIR_BASE, "scripts")
+
+
+# ====================================================================== #
+# PDB:                                                                   #
+# ====================================================================== #
+#                                                                        #
+#     /- special -/    any special pdb files, esp. the original pdb      #
```

Comment:
"special" is not descriptive. we are pointing to the idea "any PDBs that we flag for extra testing or validation, e.g. VQR, EQR, the original"